test/e2e/operator_test.go

```diff
@@ -32,6 +33,8 @@ const (
 	baseZoneDomain     = "example-test.info"
 	testNamespace      = "external-dns-test"
 	testServiceName    = "test-service"
+	testRouteName      = "test-route"
+	testRouteHost      = "myroute.example.com"
```

Comment:
I think that using a public domain for the test may be not good. Even if `myroute.example.com` doesn't seem to exist but someone can own it at some point and then it'll be always resolvable, so the test will always pass.
I'd prefer the route's host to be from the test domain you create for your test: `myroute.<commit-id>.example-test.info`.